app/pages/IdentityIssuance/NavigationBlock.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { push } from 'connected-react-router';
+import { Location } from 'history';
+import { Prompt } from 'react-router-dom';
+
+interface Props {
+    shouldPrompt: (location: Location) => boolean;
+}
+
+const message = 'You are about to abort creating an identity. Are you sure?';
+
+/**
+ * This Component is used to block navigation away from the identity issuance flow.
+ * Uses Prompt to block navigation, and Electron's own messageBox to prompt the user.
+ */
+export default function NavigationBlock({ shouldPrompt }: Props) {
+    const [blocking, setBlocking] = useState(true);
+    const dispatch = useDispatch();
+
+    const showPrompt = useCallback(
+        async (location: Location) => {
+            setBlocking(false);
+            const { response } = await window.messageBox({
+                message,
+                type: 'warning',
+                noLink: true,
+                title: 'Concordium',
```

Comment:
Since we are renaming the wallet, I think it would be better if it just matched the full name?
```suggestion
                title: 'Concordium Desktop Wallet',
```